.github/workflows/integration-testing.yaml

```diff
@@ -29,22 +29,21 @@ jobs:
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
 
-      - name: Restore cache of node modules and yarn cache
-        uses: actions/cache@v2
-        if: ${{ !env.ACT }}
-        id: nodejs-cache
+      - uses: actions/cache@v2
+        id: yarn-cache
         with:
           path: |
             node_modules
             */*/node_modules
             ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-nodejs-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
```

Comment:
is that relevant to the task?

Reply:
small bugfix because Github actions used different cache keys before